core/src/main/java/org/keycloak/representations/idm/ClientPolicyConditionRepresentation.java

```diff
@@ -32,6 +34,7 @@ public class ClientPolicyConditionRepresentation {
     private String conditionProviderId;
 
     @JsonProperty("configuration")
+    @Schema(type = SchemaType.ARRAY, implementation = String.class)
```

Comment:
Are we sure this is correct? Technically a `JsonNode` is a polymorphic type  (unfortunately).